amethyst_network/src/lib.rs

```diff
@@ -28,22 +29,25 @@ pub use {
     network_socket::NetSocketSystem,
 };
 
-use std::net::{SocketAddr, UdpSocket};
+use std::net::SocketAddr;
 
 use bincode::{deserialize, serialize, ErrorKind};
+use laminar::net::UdpSocket;
+use laminar::Packet;
 use serde::{de::DeserializeOwned, Serialize};
 
 /// Sends an event to the target NetConnection using the provided network Socket.
 /// The socket has to be bound.
-pub fn send_event<T>(event: &NetEvent<T>, target: &SocketAddr, socket: &UdpSocket)
+pub fn send_event<T>(event: &NetEvent<T>, addr: &SocketAddr, socket: &mut UdpSocket)
```

Comment:
Just a thought but this would be a good thing to turn into a trait.

```rust
trait Transport {
  type Event;
  type Error;

  fn send_event(&mut self, addr: &SocketAddr, event: &Event);

  fn receive_event(&self) -> Result<Event, Error>;
}
```